plugins/node/opentelemetry-instrumentation-express/README.md

```diff
@@ -78,6 +79,37 @@ Express instrumentation has few options available to choose from. You can set th
 - `info: ExpressRequestInfo` containing the incoming Express.js request, the current route handler creating a span and `ExpressLayerType` - the type of the handling layer or undefined when renaming the root HTTP instrumentation span.
 - `defaultName: string` - original name proposed by the instrumentation.
 
+#### Using `requestHook`
+
+Instrumentation configuration accepts a custom "hook" function which will be called on every request with the request handler span and the request information. Custom attributes can be set on the span or run any custom logic per request.
+
+Here is a simple example that adds to the request handler span some attributes based on the Express request attributes:
+
+```javascript
+import { ExpressInstrumentation, ExpressLayerType } from "@opentelemetry/instrumentation-express"
+
+const expressInstrumentation = new ExpressInstrumentation({
+  requestHook: function (
+    span: Span,
+    layerType: ExpressLayerType,
+    expressRequest: PatchedRequest
+  ) {
+
+    if (layerType === ExpressLayerType.REQUEST_HANDLER) {
+      currentSpan.setAttribute(
+        'request method',
```

Comment:
even though attribute names can contain spaces, I suggest using `http.method` and `express.base_url` to make it consistent with semantic conventions style.

formal examples and documentation should promote proper usage of attribute names and semantic conventions where possible.